lib/wp2middleman/frontmatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(post, include_fields: [])
     def post_data
       data = {
         'title' => post.title,
-        'date' => post.date_published,
+        'date' => post.date_time_published,
```

Comment:
Hmm, I don't know.  I'd be tempted to say it should safe to presume that if we're looking at files generated by WP, they'd not have validity errors.  But I don't know enough about WP to know whether it can generate invalid data in some circumstances?